website/content/v1.0/guides/kubespan.md

```diff
@@ -54,7 +54,7 @@ cluster:
 
 ### Upgrading an Existing Cluster
 
-In order to enable KubeSpan for an existing cluster, upgrade to the latest v0.14.
+In order to enable KubeSpan for an existing cluster, upgrade to the latest version.
```

Comment:
shouldn't we say the latest minimum version maybe? :shrug:

Reply:
this is docs for 1.0, so I would say update to the latest version? I'm not sure if this section should be completely removed?

Reply:
yeh, it's fine I guess, don't want someone to use an old version just because the docs said so